types/relay-runtime/lib/handlers/RelayDefaultMissingFieldHandlers.d.ts

```diff
@@ -1,3 +1,3 @@
 import { MissingFieldHandler } from '../store/RelayStoreTypes';
 
-export const missingViewerFieldHandler: MissingFieldHandler;
```

Comment:
`missingViewerFieldHandler` isn't being exported by this filer rather, tt's exporting an array with `missingViewerFieldHandler` as the only element. Not sure if this new typing is accurately representing that

Reply:
Good catch!

1. However, the upstream version doesn’t actually export a named type, rather a default…